modules/gapi/src/backends/common/serialization.cpp

```diff
@@ -292,25 +292,87 @@ I::IStream& operator >> (I::IStream& is, cv::gapi::wip::IStreamSource::Ptr &)
     return is;
 }
 
-I::OStream& operator<< (I::OStream& os, const cv::detail::VectorRef &)
+namespace
+{
+template<typename Ref, typename T, typename... Ts>
+struct putToStream;
+
+// FIXME: workaround to end the recursion
```

Comment:
This is not actually a workaround anymore, it is a valid termination point

Reply:
Removed, thanks